tests/TestCase/ORM/QueryTest.php

```diff
@@ -1617,8 +1617,7 @@ public function testCountBeforeFind(): void
         $table->getEventManager()
             ->on('Model.beforeFind', function (EventInterface $event, $query): void {
                 $query
-                    ->limit(1)
-                    ->order(['Articles.title' => 'DESC']);
```

Comment:
We might need the order to ensure results are consistently ordered, as we will eventually get an incorrect result over time if there is more than one row in the table.

Reply:
There is an issue with the count query not including tables. I was testing this change but the other failure shows it.